### PR TITLE
New migration to rinkeby

### PR DESCRIPTION
**Pull request by josojo**
Since, the driver code is already expecting that submitSolution is returning some value, this new deployment is overdue.

I want to push it and use it in AWS as well, just to test some things.

I know that we have to redeploy in just a few days again.

Function signature changes are in
```
submitsolution
cancelOrder
deleteOrder
```

### CODE REVIEW

File: networks.json [hunk 1]
```diff
@@ -3,27 +3,18 @@
...
-        "SnappBaseCore": "0x3709B8E6E2a684A1c97809E616877A6577e9e131"
+        "SnappBaseCore": "0x391F9a3A6A8752B9feF5f6b13B701c55A2514237"
       },
-      "address": "0x01Ad09b1AD496407651aa943874bC0664834BFA1",
-      "transactionHash": "0x2b814ea1d93f34719358639f5d499c62cb7a0438d5da9da84eb283477eef89e0"
-    }
-  },
-  "SnappBaseCore": {
-    "4": {
-      "links": {
```
Comment (nlordell) [inline networks.json:22]: Was it intentional to remove `SnappBaseCore` from the `networks.json`? I guess it makes sense to remove it if it isn't a public facing interface.

Comment (josojo) [reply networks.json:22]: It comes from the manual deletion of the event-outputs. Good catch

I will not do that again. If we really don't wanna have the events data in there, we should modify the js code, which does the extraction.

File: networks.json [hunk 2]
```diff
@@ -3,27 +3,405 @@
...
+      "address": "0xfb0cbD725B3970239E0437D92A04b7676992C3be",
+      "transactionHash": "0x19f272db8dd11c989d37f2459dc1c8b84ef75959e8c382b2bd1943ce8af64493"
     }
   },
   "SnappAuction": {
     "4": {
       "links": {
         "IdToAddressBiMap": "0x31172bB2B5f97E8E89cf3376495d7Bc7252f5a53",
-        "SnappBaseCore": "0x3709B8E6E2a684A1c97809E616877A6577e9e131"
+        "SnappBaseCore": "0x391F9a3A6A8752B9feF5f6b13B701c55A2514237"
```
Comment (bh2smith) [inline networks.json]: Did this address really change? Is there some way to deploy only StableX contracts?

**Review by bh2smith — APPROVED** [review]
Seems fine!

**Comment by fleupold** [comment]
We should probably look into adding an e2e test to make sure we know when we are breaking the rinkeby deployment

**Review by fleupold — APPROVED** [review]

**Review by fleupold — COMMENTED** [review]
Can you verify the contract on etherscan?

**Comment by bh2smith** [comment]
> Can you verify the contract on etherscan?

Don't we expect a new deployment before the end of the week?

**Pull request merged**